userspace/libsinsp/threadinfo.h

```diff
@@ -379,6 +371,16 @@ VISIBILITY_PRIVATE
 
 		return &(root->m_fdtable);
 	}
+
+public:
+VISIBILITY_PRIVATE
```

Comment:
I am confused by these two lines in juxtaposition. Either I don't understand `VISIBILITY_PRIVATE`, which is the most likely scenario, or this is a copy / paste weirdness

Reply:
this is one of the reasons to....not do stuff like this....

But visibility private is not

"set to private if flag is set, set to public otherwise"

It is "set to private if flag is set, don't set anything otherwise"

So by adding a protected block above this, i would change the behavior of this block when the flag is not set. so we need the extra public to prevent that.